external/dynamixel-workbench/dynamixel_workbench_controllers/src/position_control.cpp

```diff
@@ -512,29 +513,15 @@ void PositionControl::controlLoop()
     cameraStatePublish();
 }
 
-// bool PositionControl::jointCommandMsgCallback(dynamixel_workbench_msgs::JointCommand::Request &req,
-//                                               dynamixel_workbench_msgs::JointCommand::Response &res)
-// {
-//   int32_t goal_position = 0;
-//   int32_t present_position = 0;
-
-//   if (req.unit == "rad")
-//   {
-//     goal_position = dxl_wb_->convertRadian2Value(req.id, req.goal_position);
-//   }
-//   else if (req.unit == "raw")
-//   {
-//     goal_position = req.goal_position;
-//   }
-//   else
-//   {
-//     goal_position = req.goal_position;
-//   }
-
-//   bool ret = dxl_wb_->goalPosition(req.id, goal_position);
+bool PositionControl::setPIDMsgCallback(dynamixel_workbench_msgs::SetPID::Request &req,dynamixel_workbench_msgs::SetPID::Response &res)
+{
+    int joint_num = req.joint_num;
+    dxl_wb_arm_->itemWrite(dxl_id_arm_[joint_num], "Position_P_Gain", req.P);
+    dxl_wb_arm_->itemWrite(dxl_id_arm_[joint_num], "Position_I_Gain", req.I);
+    dxl_wb_arm_->itemWrite(dxl_id_arm_[joint_num], "Position_D_Gain", req.D);
 
-//   res.result = ret;
-// }
+    res.result = true;
```

Comment:
Why this is uncommented

Reply:
@ykarmesh it was commented before. I just override it